src/Model/CollectionInfoCommandIterator.php

```diff
@@ -32,6 +33,19 @@
  */
 class CollectionInfoCommandIterator extends IteratorIterator implements CollectionInfoIterator
 {
+    /** @var string|null */
+    private $databaseName;
+
+    /**
+     * @param string|null $databaseName
```

Comment:
Since the first argument is typed, it does not need a doc block to further clarify the type of the argument. I can add the `@param` annotation, but it doesn't add any value to the method.

Reply:
Oh, I didn't realize this was just intended to document the union type. SGTM.

Reply:
Yep: we can't use nullable types yet as they were only introduced in PHP 7.1.